Harvest.Web/ClientApp/src/Projects/ProjectTable.tsx

```diff
@@ -73,7 +73,13 @@ export const ProjectTable = (props: Props) => {
         accessor: "status",
         Cell: (data: Cell<Project>) => (
           <span className={`project-status-${data.row.original.status}`}>
-            {data.row.original.status}
+            {data.row.original.status
+              // Regex to insert a space before all caps
+              .replace(/([A-Z])/g, " $1")
+              // Regex to uppercase the first word
+              .replace(/^./, (str) => {
+                return str.toUpperCase();
+              })}
```

Comment:
Move this into a helper somewhere in the utils folder.  It's non-intuitive enough and it needs to be used >1 so it'll be better there.

Reply:
Gotcha, wasn't sure if we needed to do that